c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,47 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @param {angular.$templateCache} $templateCache service
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce, $templateCache) {
+  var template = function(doctype) {
+    return $templateCache.get('/static/partials/suggestion' + doctype + '.html');
+  };
+
+  return {
+    restrict: 'E',
+    scope: true,
+    /**@suppress {checkTypes} for JSON.parse */
+    link: function(scope, element, attrs) {
+
+      scope.highlight = function(text, search) {
```

Comment:
I am wondering if `highlight` might not get renamed during the compilation. Have you tested with the built version?